execution-frameworks/Invoke-AtomicRedTeam/Invoke-AtomicRedTeam.psd1

```diff
@@ -0,0 +1,123 @@
+#
+# Module manifest for module 'Invoke-AtomicRedTeam'
+#
+# Generated by: Josh Rickard
+#
+# Generated on: 09/13/2018
+#
+
+@{
+
+    # Script module or binary module file associated with this manifest.
+    RootModule        = 'Invoke-AtomicRedTeam.psm1'
+
+    # Version number of this module.
+    ModuleVersion     = '1.0.0.0'
+
+    # Supported PSEditions
+    CompatiblePSEditions = @('Desktop')
+
+    # ID used to uniquely identify this module
+    GUID              = '8f492621-18f8-432e-9532-b1d54d3e90bd'
+
+    # Author of this module
+    Author            = 'Casey Smith @subTee'
```

Comment:
While Casey is the original author, I think he would agree that you should be included given the significant improvements you've provided. Please add yourself if you're comfortable with that :).

Reply:
This is resolved.